### DISCUSSION TITLE
DUVIDA GERAL

### DISCUSSION THREAD

**Discussion by AlexArantesBRx** [RamonSilva20/mapos#1858] [General]
Eu fiz uma instalação porem a mesma não está mais funcionando no 000webhost, então vou instalar na maquina offline, porem como não abre o  site apenas consigo copiar os arquivos, como faço backup dos dados?

**Comment by Pr3d4dor** [comment]
@AlexArantesBRx No próprio mapos lá em Configurações vc pode fazer o backup do banco.

**Reply by AlexArantesBRx** [reply]
porem o mapos nao abre mais, nas pastas dele fica algum banco?

**Comment by jsfachini** [comment]
@AlexArantesBRx caso estiver usando o XAMPP > da uma olhada em **xampp\mysql\data** ( dentro dessa pasta DATA deve estar outra pasta com o nome do banco de dados criado na instalação.

**Reply by AlexArantesBRx** [reply]
na verdade está em um host online o 000webhost porem o mesmo nao da para abrir mais a pagina de banco de dados está com defeito, nao fica salvo na pasta do mapos?

**Reply by jsfachini** [reply]
Se você não tem acesso a pasta mysql mesmo sendo lá no 000webhost, ai complicou, caso não tenha nenhum bkp fora do 000webhost...

**Reply by jsfachini** [reply]
veja se te ajuda, pelo que vi você pode conseguir acesso pelo ftp 

https://youtu.be/ZaLGRGl_zqg
